guides/common/modules/proc_registering-to-red-hat-subscription-management.adoc

```diff
@@ -7,9 +7,8 @@ endif::[]
 
 Registering the host to Red Hat Subscription Management enables the host to subscribe to and consume content for any subscriptions available to the user.
 This includes content such as {RHEL} and {ProjectName}.
-ifdef::satellite[]
-For {RHEL} 7, it also provides access to Red{nbsp}Hat Software Collections (RHSCL).
-endif::[]
+
+For {RHEL} 8, it also provides access to Red{nbsp}Hat Software Collections (RHSCL).
```

Comment:
RHSCL doesn't exist on RHEL 8 so this line can be dropped.

Reply:
Addressed :+1: